network/simple-onchain-discovery/src/lib.rs

```diff
@@ -161,7 +176,11 @@ impl ConfigurationChangeListener {
             match self.conn_mgr_reqs_tx.send(update).await {
                 Ok(()) => (),
                 Err(e) => warn!(
-                    NetworkSchema::new(&self.network_context),
+                    NetworkSchema::new(&self.network_context).with_dimension_count(
+                        &DISCOVERY_COUNTS,
```

Comment:
hmm... I guess you don't need this as a formal field, since it is defined as the module in the logs

Reply:
I'm not sure I get what you mean, but I've gone ahead and split these metrics off the logs.